scripts/gen_fingerprints.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```

Comment:
Thank you for these contributions, @TheTechromancer. Would it be possible to include a comment explaining of this purpose of this file for future maintainers? It's just so to make things easier for others when they want to contribute to the project.

Reply:
Sure! Note that right now, `fingerprints.json` doesn't have all the information needed for automated detection. There are still a few steps we want to take, including expanding the table to include other fields such as the vulnerable domains involved. But this PR is meant to lay the groundwork, as those steps will take significantly more time and testing.